### ISSUE TITLE
Simply doesn't work in VS Code 1.5.3

### ISSUE THREAD

**Issue by djihbril** [abusaidm/html-snippets#14]

**Comment by abusaidm** [comment]
hello, please try update to latest vs code things should be ok there.

**Comment by djihbril** [comment]
I am now on version 1.6.0 and is still doesn't work.
Is there any additional configuration I need to make this work?
Does the file I am working in need to be a particular extension (I am working with react .jsx files)?

**Comment by abusaidm** [comment]
hello, 
yes currently this extention works on html and php files only, I will try add the support for jsx tonight or this weekend if i can.

**Comment by djihbril** [comment]
Thanks a lot! this is pretty nice of you. While you are at it, could you also add support for .js files as well? 
Sometimes you need to generate html elements from your .js files.
Again, thank you very much, I really appreciate it!

**Comment by faabiopontes** [comment]
Hey @abusaidm 
I looked into it.
Adding these lines to package.json should do the trick.
```
{
  "language": "javascript",
  "path": "./snippets/snippets.json"
},
{
  "language": "javascriptreact",
  "path": "./snippets/snippets.json"
}
```

**Comment by abusaidm** [comment]
Thanks @faabiopontes I forgot to come back and add the js and react ones like @djihbril asked.

all added and published in version 0.0.18
Thanks for your help.

If you find any other issue please let me know and will do my best to resolve it

**Comment by djihbril** [comment]
Thanks guys. Works like a charm.
👍 👍 for the prompt responses!
